src/System.Diagnostics.Tools/src/System.Diagnostics.Tools.builds

```diff
@@ -6,6 +6,9 @@
     <Project Include="System.Diagnostics.Tools.csproj">
       <TargetGroup>net46</TargetGroup>
     </Project>
+    <Project Include="System.Diagnostics.Tools.csproj">
+      <TargetGroup>netcore50</TargetGroup>
```

Comment:
Please update this to netcore50aot, as it depends on the S.P.CoreLib in the targeting pack so it can only work on that target.